src/pages/ReceiverDetails.tsx

```diff
@@ -34,6 +38,7 @@ import { renderNumberOrDash } from "helpers/renderNumberOrDash";
 import { formatDateTime } from "helpers/formatIntlDateTime";
 import { shortenAccountKey } from "helpers/shortenAccountKey";
 import { ReceiverWallet } from "types";
+import { NotificationWithButtons } from "components/NotificationWithButtons";
```

Comment:
Nit: imports should be grouped if possible (components, helpers, etc.), and types should be the last one (before relative imports).